manifests/j/Jio/JioMeet/4.22.0.30/Jio.JioMeet.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Jio.JioMeet
+PackageVersion: 4.22.0.30
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://jiomeetpro.jio.com/download/windows/JioMeetSetup.exe
+  InstallerSha256: 30CF7734AF724D6C9DD9FB817D41F4E13A8852FD099FB896783617948345C296
+  InstallerSwitches:
+    Silent: /exenoui
+    SilentWithProgress: NA

```

```suggestion
    SilentWithProgress: /exenoui
```
Pipeline is failing because `NA` is not a valid command line argument